hibike/__init__.py

```diff
@@ -0,0 +1,3 @@
+__all__ = ['hibike_simulator', 'hibike_message']
+from . import hibike_simulator
+from . import hibike_message
```

Comment:
Note the little red symbol at the last line of this diff. It means you don't have a newline at the end of file. This is typically discouraged style, because it pops up in the diff like this.